client/my-sites/sharing/connections/service-examples.jsx

```diff
@@ -1,254 +1,306 @@
+				image: {
+					src: '/calypso/images/sharing/linkedin-sharing.png',
+					alt: this.props.translate( 'Add a sharing button', { textOnly: true } )
+				},
+				label: this.props.translate(
+					'Add a {{link}}sharing button{{/link}} to your posts so readers can share your story with their connections.', {
+						components: {
+							link: <a href={ this.getSharingButtonsLink() } />
 						}
-					];
-				}
-			} )
-		};
-	},
+					}
+				),
+			},
+		];
+	}
+
+	path() {
+		return [
+			{
+				image: {
+					src: '/calypso/images/sharing/path-publicize.png',
+					alt: this.props.translate( 'Share posts to your Path timeline', { textOnly: true } )
+				},
+				label: this.props.translate( '{{strong}}Connect{{/strong}} to automatically share posts to your Path timeline.', {
+					components: {
+						strong: <strong />
+					}
+				} ),
+			},
+		];
+	}
 
-	getExamples: function() {
-		var examples;
+	tumblr() {
+		return [
+			{
+				image: {
+					src: '/calypso/images/sharing/tumblr-publicize.png',
+					alt: this.props.translate( 'Share posts to your Tumblr blog', { textOnly: true } )
+				},
+				label: this.props.translate( '{{strong}}Connect{{/strong}} to automatically share posts to your Tumblr blog.', {
+					components: {
+						strong: <strong />
+					}
+				} ),
+			},
+			{
+				image: {
+					src: '/calypso/images/sharing/tumblr-sharing.png',
+					alt: this.props.translate( 'Add a sharing button', { textOnly: true } )
+				},
+				label: this.props.translate(
+					'Add a {{link}}sharing button{{/link}} to your posts so readers can share your story with their followers.', {
+						components: {
+							link: <a href={ this.getSharingButtonsLink() } />
+						}
+					}
+				),
+			},
+		];
+	}
 
-		if ( this.props.examples[ this.props.service.ID ] ) {
-			examples = this.props.examples[ this.props.service.ID ].call( this );
+	twitter() {
+		return [
+			{
+				image: {
+					src: '/calypso/images/sharing/twitter-publicize.png',
+					alt: this.props.translate( 'Share posts to your Twitter followers', { textOnly: true } )
+				},
+				label: this.props.translate( '{{strong}}Connect{{/strong}} to automatically share posts with your Twitter followers.', {
+					components: {
+						strong: <strong />
+					}
+				} ),
+			},
+			{
+				image: {
+					src: '/calypso/images/sharing/twitter-timeline.png',
+					alt: this.props.translate( 'Add a Twitter Timeline Widget', { textOnly: true } )
+				},
+				label: this.props.translate( 'Add a {{link}}Twitter Timeline Widget{{/link}} to display your latest tweets on your site.', {
+					components: {
+						link: <a href="https://support.wordpress.com/widgets/twitter-timeline-widget/" />
+					}
+				} ),
+			},
+		];
+	}
 
-			return examples.map( function( example, i ) {
-				return <ServiceExample key={ i } image={ example.image } label={ example.label } single={ 1 === examples.length } />;
-			} );
+	render() {
+		if ( -1 === SERVICES_WHITELIST.indexOf( this.props.service.ID ) ) {
```

Comment:
Do you think it might be worth pulling in [`_.includes`](https://lodash.com/docs/4.16.4#includes) for readability sake?